cdc/owner/changefeed.go

```diff
@@ -197,6 +197,7 @@ func (c *changefeed) tick(ctx cdcContext.Context, state *orchestrator.Changefeed
 	// and we need to use the latest table names.
 	if c.currentTableNames == nil {
 		c.currentTableNames = c.schema.AllTableNames()
+		log.Debug("changefeed current table names updated", zap.Any("tables", c.currentTableNames))
```

Comment:
Could you add `changefeed` field?

Reply:
Good catch! 
Thanks for your review!  💚 💙 💜 💛 ❤️